argo/template/abtest.yaml

```diff
@@ -28,7 +25,10 @@ spec:
             -image-version={{inputs.parameters.image_version}} \
             -abtest.b.version={{inputs.parameters.b_version}} \
             -purge=true \
```

Comment:
we can also parameterize `purge`, and set true as it default value on workflow file.

Reply:
Good idea.